pyproject.toml

```diff
@@ -87,31 +87,31 @@ gunicorn = "^21.2.0"
 django-hCaptcha = "^0.2.0"
 reporters-db = "^3.2.36"
 django-waffle = "^4.0.0"
-nameparser = "^1.1.1"
-eyecite = "^2.3.0"
-scipy = "^1.11.2"
-scikit-learn = "^1.2.1"
-numpy = "^1.25.2"
+nameparser = "^1.1.3"
+eyecite = "^2.5.2"
+scipy = "^1.11.4"
+scikit-learn = "^1.3.2"
+numpy = "^1.26.2"
 datasketch = "^1.6.0"
-PyStemmer = "^2.0.1"
+PyStemmer = "^2.2.0.1"
```

Comment:
Changelog: https://github.com/snowballstem/pystemmer/blob/master/ChangeLog

Reply:
actually...this is [used](https://github.com/freelawproject/courtlistener/blob/6580ed458c52a4a10c89c944b12573c2282a8fcc/cl/citations/group_parentheticals.py#L39)...so I've re-added it

Reply:
Oh, annoying. Glad you caught that.